### PR TITLE
Transfer learning: fix model visualization to report the predicted label correctly

### PR DESCRIPTION
**Pull request by okurisama**
The current tutorial reports the validation set label as though it was the predicted label

### CODE REVIEW

File: beginner_source/transfer_learning_tutorial.py [hunk 1]
```diff
@@ -253,7 +253,9 @@ def visualize_model(model, num_images=6):
             images_so_far += 1
             ax = plt.subplot(num_images//2, 2, images_so_far)
             ax.axis('off')
-            ax.set_title('predicted: {}'.format(dset_classes[labels.data[j]]))
+            ax.set_title('predicted: {}, actual: {}'.format(
```
Comment (chsasank) [inline beginner_source/transfer_learning_tutorial.py]: Thanks for catching that! Showing both predicted and actual takes too much space. Can you please just show predicted?

File: beginner_source/transfer_learning_tutorial.py [hunk 2]
```diff
@@ -253,7 +253,7 @@ def visualize_model(model, num_images=6):
             images_so_far += 1
             ax = plt.subplot(num_images//2, 2, images_so_far)
             ax.axis('off')
-            ax.set_title('predicted: {}'.format(dset_classes[labels.data[j]]))
+            ax.set_title('predicted: {}'.format(dset_classes[preds[j]]))
```
Comment (chsasank) [inline beginner_source/transfer_learning_tutorial.py]: It has to be preds[j, 0] because torch.max leaves a dummy axis.

Comment (okurisama) [inline beginner_source/transfer_learning_tutorial.py]: Really?  It works fine like this on my machine, using current Pytorch master head.

Comment (chsasank) [inline beginner_source/transfer_learning_tutorial.py]: Yep, in the latest release, this does raise an error.

**Comment by chsasank** [comment]
#119 Fixes the same issue.

**Comment by chsasank** [comment]
Thanks for the PR though :)
I somehow missed the last commit.
